src/main/groovy/com/blackducksoftware/integration/hub/detect/util/executable/ExecutableRunner.java

```diff
@@ -74,6 +75,17 @@ public ExecutableOutput execute(final Executable executable, final boolean runQu
         }
     }
 
+    public void executeToFile(final Executable executable, final File file) throws ExecutableRunnerException {
+        logger.debug(String.format("Running executable >%s", executable.getExecutableDescription()));
+        try {
+            final ProcessBuilder processBuilder = executable.createProcessBuilder();
```

Comment:
This method should take in 2 files, one to send the standard output to and one to send the error output to.
OR
It should take in the file to send the standard output to and throw ExecutableRunnerException if the errorOutput is not empty.
We should not be ignoring the error output